python/lsst/ts/genericcamera/config_schema.py

```diff
@@ -28,10 +28,15 @@
 $schema: http://json-schema.org/draft-07/schema#
 $id: https://github.com/lsst-ts/ts_genericcamera/blob/master/python/lsst/ts/genericcamera/config_schema.py
 # title must end with one or more spaces followed by the schema version, which must begin with "v"
-title: GenericCamera v2
+title: GenericCamera v3
 description: Schema for GenericCamera configuration files
 type: object
 properties:
+  s3instance:
+    type: string
```

Comment:
Consider doing

```
anyOf:
  - type: string
  - type: "null"
```

Then you can make `s3instance = None`.

Reply:
Implemented.